[2024-03-21 18:45:01,816] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-21T18:44:00+00:00 [queued]>
[2024-03-21 18:45:01,841] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2024-03-21T18:44:00+00:00 [queued]>
[2024-03-21 18:45:01,841] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 18:45:01,841] {taskinstance.py:1088} INFO - Starting attempt 1 of 6
[2024-03-21 18:45:01,841] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 18:45:01,854] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): retrieve_raw_data> on 2024-03-21T18:44:00+00:00
[2024-03-21 18:45:01,861] {standard_task_runner.py:52} INFO - Started process 39647 to run task
[2024-03-21 18:45:01,865] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'retrieve_raw_data', '2024-03-21T18:44:00+00:00', '--job-id', '10338', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpc5j93jot', '--error-file', '/tmp/tmp98lgu3rl']
[2024-03-21 18:45:01,865] {standard_task_runner.py:77} INFO - Job 10338: Subtask retrieve_raw_data
[2024-03-21 18:45:01,912] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.retrieve_raw_data 2024-03-21T18:44:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-21 18:45:01,977] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=retrieve_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T18:44:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-21T18:44:00+00:00
[2024-03-21 18:45:02,391] {python.py:151} INFO - Done. Returned value was: None
[2024-03-21 18:45:02,409] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=retrieve_raw_data, execution_date=20240321T184400, start_date=20240321T184501, end_date=20240321T184502
[2024-03-21 18:45:02,462] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-03-21 18:45:02,480] {local_task_job.py:151} INFO - Task exited with return code 0
